next.config.js

```diff
@@ -23,6 +23,11 @@ const moduleExports = {
   images: {
     loader: 'default',
   },
+
+  env: {
+    ANALYTICS_ENABLED: process.env.ANALYTICS_ENABLED == 'true' ? 'true' : 'false',
```

Comment:
This would be cleaner:
```ts
env: {
    ANALYTICS_ENABLED: process.env.ANALYTICS_ENABLED == 'true',
}
```
So that `process.env.ANALYTICS_ENABLED` is a bool field that it's meant to be (2 states)

Reply:
True that, it is cleaner
https://github.com/LBHackney-IT/housing-repairs-online-frontend/pull/82/commits/2051e2ff9e2b2675188e8e2a443752b69446f09c